### PR TITLE
Date branch changes

### PR DESCRIPTION
**Pull request by nikunjdoshi67**
In this PR

For Date :- 

Fixed:

Date related issues, worked on edge cases where we were not able to render

Modified:

DisplayClusterData, Dashboard Table and Project Filter form for date 

For changes in UI:

Fixed:

Dashboard page by removing the static cards related to total project, node and pods

Modified:

Storybooks, routes

Removed:

Unwanted file related to the Openstack, trial files used for self testing

### CODE REVIEW

**Comment by gagansk** [comment]
Fix the CI failures.

File: src/app/Fetchdata/DisplayClusterData.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+};
+
+class Fetchdata extends React.Component<myProps, myState> {
+  constructor(myProps) {
+    super(myProps);
+
+    this.state = {
+      isLoaded: false,
+      clusterData: [],
+      api: 'https://c507295a-b340-4a31-a144-749e6fb4c08a.mock.pstmn.io/list_projects/',
```
Comment (gagansk) [inline src/app/Fetchdata/DisplayClusterData.tsx:54]: Is this a call to the postman? If so, why are we still making calls to the postman?

File: src/app/Fetchdata/fetchdata.tsx [hunk 2]
```diff
@@ -15,14 +15,17 @@ let cluster_data = [] as any;
...
     }
 
 ]
 
 let rows = [] as any;
 const Fetchdata: React.FunctionComponent<{}> = () => {
+     // mock server url
+     // const url = "https://c507295a-b340-4a31-a144-749e6fb4c08a.mock.pstmn.io/list_projects";
+     // console.log(url);
     const url = "https://c507295a-b340-4a31-a144-749e6fb4c08a.mock.pstmn.io/list_projects";
```
Comment (gagansk) [inline src/app/Fetchdata/fetchdata.tsx]: Is this for postman?

**Review by gagansk — CHANGES_REQUESTED** [review]
Why are we still using the postman api?

File: src/app/Fetchdata/fetchdata.tsx (hunk 2 above)
Comment (shakyav) [reply src/app/Fetchdata/fetchdata.tsx]: @gagansk 
fetchdata.tsx  and fetchfilterdata.tsx files are not being used currently used, there were issues with search button when using this file so we are not using it as of now. It is kept their just for reference. We can remove it from master branch and only keep it in our local repos and only keep the displayclusterdata.tsx
@nikunjdoshi67  can update the api link to be the link provided by Sai in displayclusterdata.tsx

Comment (nikunjdoshi67) [reply src/app/Fetchdata/fetchdata.tsx]: @gagansk I have resolved the merge conflicts with @shakyav

Comment (shakyav) [reply src/app/Fetchdata/fetchdata.tsx]: @gagansk  postman requests are over so you might not be able to see the data, working on to create another server

Comment (gagansk) [reply src/app/Fetchdata/fetchdata.tsx]: @shakyav Yes. I got this message 
**Failed to load resource: the server responded with a status of 429 (Too Many Requests)**
Let us discuss moving to the paid Postman service.

**Comment by gagansk** [comment]
The default date is not coming up. Also, there is an unresponsive calendar in the date selection option. You can take it out.

Screen Shot 2020-06-11 at 3 32 23 PM

**Comment by shakyav** [comment]
@gagansk 
this PR is also working with backend
image

On date selection, time duration is 7 hrs 
image

image

**Comment by gagansk** [comment]
LGTM

**Review by gagansk — APPROVED** [review]

**Pull request merged**